outputs.tf

```diff
@@ -9,10 +14,15 @@ output "host" {
 }
 
 output "dns_name" {
-  value       = "${aws_efs_file_system.default.id}.efs.${var.aws_region}.amazonaws.com"
+  value       = "${aws_efs_file_system.default.dns_name}"
   description = "DNS name"
 }
 
+output "mount_target_dns_names" {
+  value       = "${aws_efs_mount_target.default.*.dns_name}"
```

Comment:
```suggestion
  value       = "[${aws_efs_mount_target.default.*.dns_name}"]
```

Reply:
@johncblandii  sorry, I missed this.
Since we use `[]` in all other outputs (e.g. `network_interface_ids`), let's make it consistent.

Reply:
Done